desktop/src/main/java/haveno/desktop/main/funds/withdrawal/WithdrawalView.java

```diff
@@ -175,6 +163,11 @@ public void initialize() {
             }).start();
         });
 
+        sendMaxButton.setOnAction(event -> {
+            amount = xmrWalletService.getAvailableBalance();
+            amountTextField.setText(Res.get("funds.withdrawal.maximum", Res.getBaseCurrencyCode()));
```

Comment:
Remove `Res.getBaseCurrencyCode()` since the translation does not have {0} variable.